src/Orchard.Environment.Extensions/ExtensionManager.cs

```diff
@@ -256,6 +256,11 @@ public Task<ExtensionEntry> LoadExtensionAsync(IExtensionInfo extensionInfo)
 
         public async Task<IEnumerable<FeatureEntry>> LoadFeaturesAsync()
         {
+            // for testing
```

Comment:
But at the same time, the code below is just using an unbound number of concurrent threads